Resources/Prototypes/InventoryTemplates/monkey_inventory_template.yml

```diff
@@ -61,4 +61,6 @@
     whitelist:
       tags:
         - SuitEVA
-
+    blacklist:
+      tags:
+        - NotMonkeyWearable
```

Comment:
couldn't you just remove this and add MonkeyWearable to the eva and syndie suits or am i missing something?

Reply:
ERT and bloodred medic suit inherit from normal bloodred which makes them also monkey wearable, which is unintended so that tag is added to the two suits